libmscore/glissando.cpp

```diff
@@ -370,6 +380,21 @@ void Glissando::write(Xml& xml) const
       if (_showText && !_text.isEmpty())
             xml.tag("text", _text);
       xml.tag("subtype", int(_glissandoType));
+      writeProperty(xml, P_ID::PLAY_GLISSANDO);
+      switch ( glissandoStyle()) {
+          case MScore::GlissandoStyle::BLACK_KEYS:
```

Comment:
Also an array to map the "name" of the style with the Enum value could be better no? and  typeToName / nameToType functions